sdk/spring/spring-cloud-azure-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -1568,6 +1568,13 @@
       "type": "java.lang.String",
       "description": "Name of the Azure cloud to connect to.",
       "sourceType": "com.azure.spring.cloud.service.implementation.passwordless.AzurePasswordlessProperties"
+    },
+    {
+      "name": "spring.datasource.azure.passwordless-enabled",
+      "type": "java.lang.Boolean",
+      "description": "Whether to enable supporting Azure identity token credentials.",
+      "sourceType": "com.azure.spring.cloud.service.implementation.passwordless.AzurePasswordlessProperties",
```

Comment:
Do you think we can skip this sourceType here?

Reply:
If users can still jump to AzurePasswordlessProperties from IDE, then I think we can remove it.

Reply:
But what can the users benefit from the jumping?

Reply:
> But what can the users benefit from the jumping?

The point here is what's the expectation of users, I think by default users will have the expectation that it would be able to jump to a property class.